src/apps/resilientwebdesign.svg

```diff
@@ -0,0 +1,11 @@
+        </g>
+    </g>
+</svg>
```

Comment:
@adactio Would you mind replacing this with a minified version? Running this through [SVGOMG](https://jakearchibald.github.io/svgomg/) with default settings yields:

```svg
<svg width="512" height="512" viewBox="0 0 512 512" xmlns="http://www.w3.org/2000/svg" style="background:#5F7995"><title>Resilient Web Design</title><path d="M112.72 165.28c0-5.76-2.88-11.76-6.84-15-4.8-3.96-11.28-5.04-19.56-5.04-6.48 0-15 .6-19.44.6h-6.72c-2.64 0-3.24.6-3.24 1.44 0 .96 1.08 1.56 2.88 2.04 5.4 1.32 6.48 1.56 6.96 6 .48 4.68.72 15.6.72 34.08 0 7.2-.48 19.92-.72 23.64-.36 4.56-2.52 5.28-7.8 6.96-1.56.6-2.64.84-2.64 1.8 0 1.2 1.56 1.68 3.12 1.68 1.32 0 7.56-.48 9.72-.48 8.16 0 10.44.72 15.48.72.6 0 2.88-.36 2.88-1.56 0-.6-.84-1.44-2.88-2.04-5.64-1.32-6.96-3-7.2-8.04-.36-6.96-.6-22.2-.48-24.24 0-.36.6-1.44 1.44-1.44 3.6 0 8.04 1.08 8.52 1.56.72.48 3.12 3.48 4.8 6 4.8 7.56 10.92 16.8 15.12 22.56 3.48 4.44 10.32 7.8 15.96 7.8 2.52 0 5.16-.24 6.6-.84.72-.36 1.68-.84 1.68-1.56 0-.48-.96-1.44-1.92-1.68-5.28-1.08-8.52-3.24-11.76-7.08-7.32-8.16-17.4-24.36-18.6-27.24-.24-.48.12-1.56.48-1.8.84-.6 3.12-1.68 3.72-2.04 5.88-3.6 9.72-10.32 9.72-16.8zm-10.68.24c0 7.56-4.68 12.84-7.92 14.88-1.68 1.08-8.04 3.12-15 3.12-.72 0-1.68-.84-1.92-1.8-.36-2.04 0-25.2.36-29.4 0-.48.84-2.28 1.44-2.52 1.32-.6 5.16-.84 5.76-.84 3.84 0 9 1.8 11.88 4.56 3.48 3.48 5.4 9.12 5.4 12zm55.48 9.12c-2.04 0-6 .72-9.12 2.16-9.36 4.32-14.4 13.92-14.4 23.52 0 14.04 8.64 23.88 21.6 23.88 6.84 0 13.2-2.88 17.4-9 .36-.6.84-1.92.84-2.88 0-.6-.24-1.44-.96-1.44-.48 0-1.08.72-1.56 1.44-2.04 3-5.64 4.32-10.92 4.32-5.76 0-10.8-3-13.44-6.6-2.64-3.48-4.56-9-4.56-13.68 0-1.2.24-3.24.6-4.2.12-.36.6-.96.96-.96 5.52.12 26.04.12 27.36.12 1.32 0 2.28-1.32 2.28-2.04 0-4.2-1.56-7.56-4.2-10.08-3-3-7.68-4.56-11.88-4.56zm-1.2 3.12c4.32 0 7.92 3.84 7.92 7.2 0 2.4-1.2 3-4.08 3-6 .12-12.36.12-15.48 0-.24 0-.72-.72-.48-1.32 2.16-5.52 6.72-8.88 12.12-8.88zm43.96-3.24c-4.08 0-7.32 1.08-9.6 2.52-4.56 2.76-6.6 6.72-6.6 10.8 0 5.64 3.84 9.84 7.44 12.12 3.72 2.4 4.44 3 7.32 5.04 3.72 2.28 5.4 4.8 5.4 8.04 0 2.28-.96 5.16-2.76 6.6-1.32.96-3.24 1.8-4.8 1.8-4.32 0-7.56-2.4-10.92-8.52-.72-1.32-1.56-2.76-2.52-2.76-.72 0-.96 1.2-.96 2.52 0 3.48 1.44 8.52 1.8 9.12 2.04 1.68 8.04 2.64 11.64 2.64 3.72 0 7.92-1.32 10.08-2.76 3-2.16 6.96-6.24 6.96-11.76 0-5.4-2.64-9.48-9.36-13.44-8.04-4.44-11.88-7.56-11.88-11.64 0-2.52 2.28-7.44 7.2-7.44 3.96 0 6.96 2.4 9.6 8.04.6 1.32 1.32 2.04 1.8 2.04.96 0 1.44-1.08 1.44-2.16 0-.84-.36-4.92-.36-7.08 0-.72-.72-1.56-.96-1.68-.24-.12-2.52-1.08-3.84-1.32-1.32-.36-3.84-.72-6.12-.72zm22.72 5.64c0 .48.24.84.96 1.32 3.48 2.52 4.8 3.48 4.92 4.32.24 1.44.12 18.48.12 18.72 0 3.48-.12 10.68-.48 12.36-.24 1.2-2.04 2.64-4.8 3.36-2.04.6-2.88 1.2-2.88 1.92 0 .84 1.2 1.32 3.12 1.32 2.04 0 6.24-.48 10.2-.48 2.52 0 8.52.36 9.36.36 1.32 0 2.76-.48 2.76-1.2 0-.84-.72-1.44-2.64-2.04-3-.96-5.64-1.2-5.76-5.04-.12-4.2-.24-9.24-.24-15.96 0-6.84.72-22.44.72-23.76 0-.12-.6-1.32-1.08-1.32-1.8.48-7.44 3.12-11.88 4.08-1.32.24-2.4 1.2-2.4 2.04zm10.92-30c-1.68 0-3.48.48-4.68 1.56-.96 1.32-1.44 3.24-1.44 4.32 0 2.04.84 3.36 1.8 4.44 1.08.96 3 1.44 3.84 1.44 2.4 0 5.76-2.64 5.76-5.88 0-2.16-2.04-5.88-5.28-5.88zm19.96-6.36c0 .48.36.84 2.4 2.28 2.76 1.68 2.88 2.4 2.88 3.36 0 3.6.36 16.2.36 25.08 0 18.96-.6 31.92-.84 36.48-.36 6-.6 8.16-6.12 9.36-1.32.24-2.64.84-2.64 1.68 0 .96.96 1.32 3.36 1.32 2.52 0 7.8-.36 10.56-.36 2.16 0 7.32.36 10.2.36 1.44 0 3.12-.48 3.12-1.32 0-.72-.96-1.32-2.76-1.8-5.16-1.44-5.64-2.88-6.12-5.28-.6-3.24-.36-26.16-.36-43.68 0-16.08.72-29.64 1.08-32.64 0-.48-.72-1.44-1.08-1.2-3.24 2.04-6.84 2.76-12.36 4.68-1.32.36-1.68 1.2-1.68 1.68zm31.48 36.36c0 .48.24.84.96 1.32 3.48 2.52 4.8 3.48 4.92 4.32.24 1.44.12 18.48.12 18.72 0 3.48-.12 10.68-.48 12.36-.24 1.2-2.04 2.64-4.8 3.36-2.04.6-2.88 1.2-2.88 1.92 0 .84 1.2 1.32 3.12 1.32 2.04 0 6.24-.48 10.2-.48 2.52 0 8.52.36 9.36.36 1.32 0 2.76-.48 2.76-1.2 0-.84-.72-1.44-2.64-2.04-3-.96-5.64-1.2-5.76-5.04-.12-4.2-.24-9.24-.24-15.96 0-6.84.72-22.44.72-23.76 0-.12-.6-1.32-1.08-1.32-1.8.48-7.44 3.12-11.88 4.08-1.32.24-2.4 1.2-2.4 2.04zm10.92-30c-1.68 0-3.48.48-4.68 1.56-.96 1.32-1.44 3.24-1.44 4.32 0 2.04.84 3.36 1.8 4.44 1.08.96 3 1.44 3.84 1.44 2.4 0 5.76-2.64 5.76-5.88 0-2.16-2.04-5.88-5.28-5.88zm42.52 24.48c-2.04 0-6 .72-9.12 2.16-9.36 4.32-14.4 13.92-14.4 23.52 0 14.04 8.64 23.88 21.6 23.88 6.84 0 13.2-2.88 17.4-9 .36-.6.84-1.92.84-2.88 0-.6-.24-1.44-.96-1.44-.48 0-1.08.72-1.56 1.44-2.04 3-5.64 4.32-10.92 4.32-5.76 0-10.8-3-13.44-6.6-2.64-3.48-4.56-9-4.56-13.68 0-1.2.24-3.24.6-4.2.12-.36.6-.96.96-.96 5.52.12 26.04.12 27.36.12 1.32 0 2.28-1.32 2.28-2.04 0-4.2-1.56-7.56-4.2-10.08-3-3-7.68-4.56-11.88-4.56zm-1.2 3.12c4.32 0 7.92 3.84 7.92 7.2 0 2.4-1.2 3-4.08 3-6 .12-12.36.12-15.48 0-.24 0-.72-.72-.48-1.32 2.16-5.52 6.72-8.88 12.12-8.88zm26.92 2.52c0 .36.72.96 2.28 2.04 2.64 2.16 2.88 3.36 2.88 4.68 0 12.24-.24 22.68-.6 26.28-.36 4.32-1.08 5.52-5.04 6.72-1.32.36-2.52.96-2.52 1.8 0 1.08 1.32 1.56 3.12 1.56 2.04 0 5.52-.36 9.96-.36 2.52 0 5.88.12 7.32.12 1.68 0 3-.6 3-1.44 0-.72-.96-1.44-2.16-1.8-3.84-1.08-4.32-2.04-4.44-3.72-.12-1.44-.12-27.24-.12-28.8 0-1.08.84-2.52 1.2-2.88 2.28-2.04 5.4-3.84 10.68-3.84 2.76 0 6.36 1.32 7.68 2.28 1.44 1.08 3.6 4.56 3.6 8.88 0 3.48-.12 18-.36 21.96-.24 3.96-.84 4.92-3.72 6-1.44.48-2.76 1.2-2.76 2.04 0 .96 1.68 1.44 2.76 1.44 1.56 0 5.28-.24 8.28-.24 3.36 0 6.12.24 8.88.24 1.92 0 3.6-.48 3.6-1.32 0-.84-.72-1.44-3-2.04-3-.84-4.2-2.4-4.44-4.2-.12-1.68-.36-19.56-.36-21.84 0-6-1.56-11.52-4.2-14.64-1.68-1.92-5.04-4.56-10.92-4.56-5.04 0-11.28 3.36-16.08 6.6-.12 0-.72.24-.72-.36 0-1.08.48-3.96.48-6 0-.48-.6-1.8-1.2-1.8s-1.68.6-2.88 1.32c-1.68 1.2-6 2.76-8.04 3.48-.84.36-2.16 1.56-2.16 2.4zm76.84-4.08c-.48 0-.96-.6-.96-.96v-5.64c0-.24-1.08-1.32-1.68-1.32-.48 0-1.32.96-2.76 2.64-1.68 2.04-5.4 4.92-9.6 8.28-.36.36-.72.84-.72 1.32 0 .36.72.96 1.68 1.32 2.4.72 4.56 1.56 4.56 2.4 0 4.2-.48 22.2-.48 26.64 0 3.36.24 5.88 1.56 7.92 1.44 2.64 5.64 5.64 10.56 5.64 5.4 0 9.6-2.16 12.84-5.28.84-.84 1.68-2.04 1.68-3 0-1.08-.6-1.32-.72-1.32-.48 0-.72.12-1.8.72-1.8 1.08-4.32 1.8-5.88 1.8-2.52 0-5.04-.24-7.32-2.28-1.32-1.32-2.04-2.88-2.04-5.88 0-5.76.12-23.4.24-25.8 0-.24.48-.84 1.08-.84h13.08c.84 0 3.12-.24 3.12-1.32 0-1.56-.36-4.92-.36-4.56 0-.36-1.2-.96-1.32-.96-.48 0-12.12.48-14.76.48zM61.1 290c-2.2 0-5.1-.4-6.8-.4-1.9 0-2.9.3-2.9 1.2 0 .6 1.1 1.2 2.5 1.5 2.8.7 4 3 5.6 6.7 2.1 4.9 7.9 19.4 8.6 21.4 1.5 4.4 2.8 8.1 3 8.6.4 1.2.9 1.5 1.8 1.5s2.1-3 2.5-4.1c3-7.5 6.7-15.2 9.1-20.2.1-.3.6-.3.7-.1 1.7 4.1 4.9 12.1 6.2 15.3 1 2.4 2.2 6.3 2.6 7.4.6.8 1.4 1.5 1.9 1.5s1-.2 1.3-1.2c1-3.3 2.5-6.6 4.2-10.8 3-7.4 6.3-15.6 9.6-21.5 1.6-2.7 2-3.8 5.3-4.8 1.2-.3 2.3-.6 2.3-1.5 0-.6-.9-1-2.1-1-1.6 0-4.1.5-7.4.5-1.5 0-5.1-.3-6.2-.3-1 0-2.3.4-2.3 1 0 .9 1.4 1.3 2.3 1.6 3.7.9 3.9 2.1 3.9 3.1 0 .9-.6 3.1-1.7 6-2 5.1-5 13.1-6.7 16.7-.3.6-.7.8-.9.1-2.1-4.9-7.1-18.5-7.7-20-.7-1.9-1.2-2.9-1.2-3.6 0-.4.3-1.6 2.9-2.2 1.7-.3 2.4-.9 2.4-1.6 0-.6-.9-1.1-2.4-1.1-2.2 0-5.2.3-7 .3-2.3 0-5.3-.3-6.7-.3-1.7 0-2.4.3-2.4 1 0 .8.9 1.2 2.2 1.6 2.3.6 3.1 2.2 4.2 4.2.8 1.5 1.5 4.1 1.6 4.8 0 .4 0 .9-.5 2-2 4.5-5.9 13-7 15.2-.2.5-.6.6-.9 0-.7-1.2-7.4-18.3-8.6-22.2-.9-2.3-.4-3.5 1.8-3.9 1.7-.4 2.5-1 2.5-1.6 0-.5-.8-1.1-2.6-1.1-1.7 0-4.5.3-7 .3zm79.2-1.3c-1.7 0-5 .6-7.6 1.8-7.8 3.6-12 11.6-12 19.6 0 11.7 7.2 19.9 18 19.9 5.7 0 11-2.4 14.5-7.5.3-.5.7-1.6.7-2.4 0-.5-.2-1.2-.8-1.2-.4 0-.9.6-1.3 1.2-1.7 2.5-4.7 3.6-9.1 3.6-4.8 0-9-2.5-11.2-5.5-2.2-2.9-3.8-7.5-3.8-11.4 0-1 .2-2.7.5-3.5.1-.3.5-.8.8-.8 4.6.1 21.7.1 22.8.1 1.1 0 1.9-1.1 1.9-1.7 0-3.5-1.3-6.3-3.5-8.4-2.5-2.5-6.4-3.8-9.9-3.8zm-1 2.6c3.6 0 6.6 3.2 6.6 6 0 2-1 2.5-3.4 2.5-5 .1-10.3.1-12.9 0-.2 0-.6-.6-.4-1.1 1.8-4.6 5.6-7.4 10.1-7.4zm63.3 16.8c0-5.7-2.1-11.3-6.6-15.1-3.2-2.8-7.7-4.4-12.4-4.4-3.3 0-8.1 1.2-11.4 3.4-.1.1-.6-.2-.6-.4-.2-1.1-.2-6-.2-8.6 0-8 .6-17.5.9-24.1 0-.4-.7-1.3-1.1-1.3-.2 0-1.2.4-2.5 1.2-2 1.2-5.4 2.1-8.2 3.1-.3.1-1.2 1.3-1.2 1.4 0 .5.5.8 2 1.9 1.9 1.6 2.4 2.9 2.5 3.4.2 1.4.5 16.2.5 25.7 0 8.2-.1 25.4-.6 33.2 0 .7.2 1.3.6 1.3.2 0 .8-.1 1.2-.3.8-.4 1.7-.5 2.1-.3 2.3.8 9.2 2 12.7 2s9.5-1.1 13.6-4.5c4.4-3.4 8.7-9.3 8.7-17.6zm-8.3 2.1c0 4.5-1.8 10.7-4.6 13.4-2.3 2.1-5 3.4-6.1 3.4-1.8 0-6.5-1.3-9.3-4.2-1.3-1.3-1.9-2.9-2-5-.5-5.4-.8-10.8-.8-16 0-2.1.1-4.9.2-5.8.1-.7.5-1.3.8-1.5.8-.6 2.7-1.5 6-1.5 4.7 0 8.6 2 10.9 4.3 3.4 3.7 4.9 9.3 4.9 12.9zm59.6-21.6c-2.7 0-8.6 1-13.2 3.8-4.9 2.9-10.6 8.6-10.6 18.4 0 5 1.6 10.1 4.8 13.7 3.1 3.3 7.7 5.5 12.5 5.5 3.3 0 8.4-.9 13-5 .2 0 .6.2.6.6 0 1.4-.2 3.2-.2 4.1 0 .5.7 1.3 1.1 1.3.5 0 1.3-.2 3.6-1 1.9-.7 7.1-1.7 9-2 .5-.1 1.7-1.8 1.7-2.5 0-.1-.2-.6-.6-.6-1 .1-4.3.8-5.4.8-.6 0-1.4-.4-1.5-4.1-.1-1.9-.3-2.6-.3-22.8 0-30.6.4-34.2.9-39.5.1-.9-.5-1.6-.8-1.6-.4 0-.7.1-1.3.3-2.6 1-7.9 2.5-10.2 2.8-.6.1-1.7 1.3-1.7 1.7 0 .2.1.4 1 .9 4.6 2.4 4.6 3.6 4.7 4.1.3 2.4.6 17.7.1 20.4-.1.5-.4 1.2-.8 1.1-1.2-.2-2.8-.4-6.4-.4zm-3.7 3.3c2 0 5.8 1.1 8.6 3.9 1.2 1.2 2.1 3 2.2 3.8.3 4.7.4 10.2.3 16.6-.1 3.2-1.4 6.4-1.6 6.6-1.8 1.3-4.1 2.4-7.3 2.4-8.6 0-14.2-9.2-14.2-17.2 0-5.6 2.3-10.7 4.2-12.6 1.8-1.8 4.6-3.5 7.8-3.5zm50.1-3.2c-1.7 0-5 .6-7.6 1.8-7.8 3.6-12 11.6-12 19.6 0 11.7 7.2 19.9 18 19.9 5.7 0 11-2.4 14.5-7.5.3-.5.7-1.6.7-2.4 0-.5-.2-1.2-.8-1.2-.4 0-.9.6-1.3 1.2-1.7 2.5-4.7 3.6-9.1 3.6-4.8 0-9-2.5-11.2-5.5-2.2-2.9-3.8-7.5-3.8-11.4 0-1 .2-2.7.5-3.5.1-.3.5-.8.8-.8 4.6.1 21.7.1 22.8.1 1.1 0 1.9-1.1 1.9-1.7 0-3.5-1.3-6.3-3.5-8.4-2.5-2.5-6.4-3.8-9.9-3.8zm-1 2.6c3.6 0 6.6 3.2 6.6 6 0 2-1 2.5-3.4 2.5-5 .1-10.3.1-12.9 0-.2 0-.6-.6-.4-1.1 1.8-4.6 5.6-7.4 10.1-7.4zm36.4-2.7c-3.4 0-6.1.9-8 2.1-3.8 2.3-5.5 5.6-5.5 9 0 4.7 3.2 8.2 6.2 10.1 3.1 2 3.7 2.5 6.1 4.2 3.1 1.9 4.5 4 4.5 6.7 0 1.9-.8 4.3-2.3 5.5-1.1.8-2.7 1.5-4 1.5-3.6 0-6.3-2-9.1-7.1-.6-1.1-1.3-2.3-2.1-2.3-.6 0-.8 1-.8 2.1 0 2.9 1.2 7.1 1.5 7.6 1.7 1.4 6.7 2.2 9.7 2.2 3.1 0 6.6-1.1 8.4-2.3 2.5-1.8 5.8-5.2 5.8-9.8 0-4.5-2.2-7.9-7.8-11.2-6.7-3.7-9.9-6.3-9.9-9.7 0-2.1 1.9-6.2 6-6.2 3.3 0 5.8 2 8 6.7.5 1.1 1.1 1.7 1.5 1.7.8 0 1.2-.9 1.2-1.8 0-.7-.3-4.1-.3-5.9 0-.6-.6-1.3-.8-1.4-.2-.1-2.1-.9-3.2-1.1-1.1-.3-3.2-.6-5.1-.6zm18.7 4.7c0 .4.2.7.8 1.1 2.9 2.1 4 2.9 4.1 3.6.2 1.2.1 15.4.1 15.6 0 2.9-.1 8.9-.4 10.3-.2 1-1.7 2.2-4 2.8-1.7.5-2.4 1-2.4 1.6 0 .7 1 1.1 2.6 1.1 1.7 0 5.2-.4 8.5-.4 2.1 0 7.1.3 7.8.3 1.1 0 2.3-.4 2.3-1 0-.7-.6-1.2-2.2-1.7-2.5-.8-4.7-1-4.8-4.2-.1-3.5-.2-7.7-.2-13.3 0-5.7.6-18.7.6-19.8 0-.1-.5-1.1-.9-1.1-1.5.4-6.2 2.6-9.9 3.4-1.1.2-2 1-2 1.7zm9.1-25c-1.4 0-2.9.4-3.9 1.3-.8 1.1-1.2 2.7-1.2 3.6 0 1.7.7 2.8 1.5 3.7.9.8 2.5 1.2 3.2 1.2 2 0 4.8-2.2 4.8-4.9 0-1.8-1.7-4.9-4.4-4.9zm32.8 19.9c-3.9 0-7.4 1.1-10.2 3.2-3.9 2.9-6.1 6.5-6.1 11.4 0 7.3 4.8 11.1 7.6 12.7.3.3.6.7.4.9-1.6 1.6-5.9 4-7.4 5.2-.2.2-.5.7-.5.9 0 3.2 1.4 6 3.4 7.4 1.3 1 3.2 1.6 3.9 2 .3.2.5.5.3.7-1.3 1.1-4.4 3.3-6.9 5.4-.9.7-2.9 1.9-2.9 6.3 0 2.8 1.8 6.5 4.6 8.7 4.2 3.1 8.9 3.7 12.5 3.7 3.6 0 8.9-.7 14.4-4.3 4.3-2.8 9.4-7.9 9.4-15.4 0-2.6-1.4-6.4-4.2-8.8-1.8-1.4-4-2.3-4.8-2.4-3.2-.5-14-.6-18.4-.8-1.1-.1-4.1-1.9-4.1-4.5 0-.8 2.2-2.9 3.2-3.4.4-.2.9-.2 1.5-.1.7 0 2.9.1 3.5.1 2.4 0 6.5-.4 10.9-3.8 3.9-3 5.6-7.1 5.6-12.6 0-1.7-.3-3.5-.8-5-.1-.3.2-.8.6-.8 2.5.2 6.1.2 6.7.2.4 0 1.8-1.8 1.8-3.2 0-.4-.3-1-.9-1.1-2.1-.5-9.4-.6-11.6-.8-.7-.1-1.7-.4-2.8-.6-2.9-.7-4.7-1.2-8.7-1.2zm-.8 2.4c2.9 0 5.3 1.6 7.2 4.4 1.6 2.4 2.2 5.7 2.2 7.5 0 4.1-1.3 9-3.8 10.6-1.4.9-2.6 1.4-3.3 1.4-6.5 0-9.9-7.5-9.9-12.5 0-4.7 1.1-7.3 2.6-8.9.9-1.2 3.2-2.5 5-2.5zm3.2 61.9c-3.7 0-7.1-1-9.8-3.1-2.1-1.8-3.5-4.4-3.5-7.6 0-2.3 1.3-6 4.5-8.2.6-.4 2.2-1 5.4-1 4.2 0 7.4 0 10.1.5 2.4.5 4.3 1.8 5.3 3.1 1.2 1.6 1.4 3.9 1.4 5.2 0 2.9-1.5 6.7-5.5 9.3-1.1.6-4 1.8-7.9 1.8zm24.6-59.1c0 .3.6.8 1.9 1.7 2.2 1.8 2.4 2.8 2.4 3.9 0 10.2-.2 18.9-.5 21.9-.3 3.6-.9 4.6-4.2 5.6-1.1.3-2.1.8-2.1 1.5 0 .9 1.1 1.3 2.6 1.3 1.7 0 4.6-.3 8.3-.3 2.1 0 4.9.1 6.1.1 1.4 0 2.5-.5 2.5-1.2 0-.6-.8-1.2-1.8-1.5-3.2-.9-3.6-1.7-3.7-3.1-.1-1.2-.1-22.7-.1-24 0-.9.7-2.1 1-2.4 1.9-1.7 4.5-3.2 8.9-3.2 2.3 0 5.3 1.1 6.4 1.9 1.2.9 3 3.8 3 7.4 0 2.9-.1 15-.3 18.3-.2 3.3-.7 4.1-3.1 5-1.2.4-2.3 1-2.3 1.7 0 .8 1.4 1.2 2.3 1.2 1.3 0 4.4-.2 6.9-.2 2.8 0 5.1.2 7.4.2 1.6 0 3-.4 3-1.1 0-.7-.6-1.2-2.5-1.7-2.5-.7-3.5-2-3.7-3.5-.1-1.4-.3-16.3-.3-18.2 0-5-1.3-9.6-3.5-12.2-1.4-1.6-4.2-3.8-9.1-3.8-4.2 0-9.4 2.8-13.4 5.5-.1 0-.6.2-.6-.3 0-.9.4-3.3.4-5 0-.4-.5-1.5-1-1.5s-1.4.5-2.4 1.1c-1.4 1-5 2.3-6.7 2.9-.7.3-1.8 1.3-1.8 2z" fill="#FFF" fill-rule="evenodd"/></svg>
```

Feel free to copy-paste that.